### PR TITLE
Naming convention

### PR DESCRIPTION
**Pull request by Vidushi-GitHub**
A line is added for topics names convention. Feature flag is not added, can do if required for production.

### CODE REVIEW

File: app/routes/docs.notices.producers.mdx [hunk 1]
```diff
@@ -33,8 +33,8 @@ also happy to work with the mission teams to help construct your alerts.
       <ProcessListHeading type="h3">
          Name Your Kafka Topics
       </ProcessListHeading>
-      Names of Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<i>notice_type</i></code>.
-      Pick a prefix for your Kafka topic names, <code><i>mission</i>.\*</code>.
+      The naming convention for Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<i>NoticesType</i></code>. The mission name should be lower case, while the Kafka topics should be in [CamelCase](https://en.wikipedia.org/wiki/Camel_case) format.
+      Pick a prefix for your Kafka topic names, <code><i>mission</i>.\*</code>.
```
Comment (jracusin) [inline app/routes/docs.notices.producers.mdx:37]: We shouldn't link to wikipedia.  Also, what about gcn.notices.mission.instrument.NoticeType?  Instrument is optional. And NoticesType should be NoticeType

Comment (Vidushi-GitHub) [reply app/routes/docs.notices.producers.mdx:37]: Ok, other reference sites are worse and have advertisements. "instrument.NoticeType" extra instrument word is my mistake. 
I was trying to keep Notices as a consistent word for GCN Notices.

Comment (lpsinger) [reply app/routes/docs.notices.producers.mdx:37]: > We shouldn't link to wikipedia.

Why not?

Comment (lpsinger) [inline app/routes/docs.notices.producers.mdx:37]:
```suggestion
      Pick a prefix for your Kafka topic names, <code><i>mission</i>.\*</code>.
```

**Comment by Vidushi-GitHub** [comment]
Closing this, as we have decided to change schema name convention. So topic names will remain intact.

File: app/routes/docs.notices.producers.mdx (hunk 1 above)
Comment (jracusin) [reply app/routes/docs.notices.producers.mdx]: It's a NASA policy when it comes to public communications to only link to trusted other agency or institution websites.  Anyone can change the content of wikipedia.

Comment (lpsinger) [reply app/routes/docs.notices.producers.mdx]: > It's a NASA policy when it comes to public communications to only link to trusted other agency or institution websites. Anyone can change the content of wikipedia.

Is there a link to that policy? I haven't heard of it before.

**Comment by lpsinger** [comment]
@Vidushi-GitHub, it seems that we have decided that notice types will be lower case with underscores, so this still needs to be fixed. Would you please reopen this and update it?

File: app/routes/docs.notices.producers.mdx [hunk 2]
```diff
@@ -33,7 +33,7 @@ also happy to work with the mission teams to help construct your alerts.
       <ProcessListHeading type="h3">
          Name Your Kafka Topics
       </ProcessListHeading>
-      Names of Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<i>notice_type</i></code>.
+      The naming convention for Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<i>NoticesType</i></code>. The mission name should be lower case, while the Kafka topics should be in [CamelCase](https://en.wikipedia.org/wiki/Camel_case) format.
```
Comment (Vidushi-GitHub) [inline app/routes/docs.notices.producers.mdx:36]:
```suggestion
      The naming convention for Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<I>notices_type</i></code>. The mission name should be in lowercase, and the Kafka topics should be in snake_case format, with the words in lowercase separated by underscores. Example for a single observatory, such as IceCube is <code>gcn.notices.<I>icecube</i>.<i>lvk_nu_track_search</i></code>. For missions with multiple instruments, you can include the instrument name as <code>gcn.notices.<i>mission</i>.<I>instrument</i>.<I>notices_type</i></code>, for example, Swift-BAT Kafka topic should be <code>gcn.notices.<I>swift</i>.<I>bat</i>.<I>alert</i></code>.
```

File: app/routes/docs.notices.producers.mdx [hunk 3]
```diff
@@ -33,7 +33,7 @@ also happy to work with the mission teams to help construct your alerts.
       <ProcessListHeading type="h3">
          Name Your Kafka Topics
       </ProcessListHeading>
-      Names of Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<i>notice_type</i></code>.
+      The naming convention for Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<I>notices_type</i></code>. The mission name should be in lowercase, and the Kafka topics should be in snake_case format, with the words in lowercase separated by underscores. Example for a single observatory, such as IceCube is <code>gcn.notices.<I>icecube</i>.<i>lvk_nu_track_search</i></code>. For missions with multiple instruments, you can include the instrument name as <code>gcn.notices.<i>mission</i>.<I>instrument</i>.<I>notices_type</i></code>, for example, Swift-BAT Kafka topic should be <code>gcn.notices.<I>swift</i>.<I>bat</i>.<I>alert</i></code>.
```
Comment (lpsinger) [inline app/routes/docs.notices.producers.mdx:36]:
```suggestion
      The naming convention for Kafka topics follow the format <code>gcn.notices.<i>mission</i>.<i>notices_type</i></code>. The mission name should be in lowercase, and the Kafka topics should be in snake_case format, with the words in lowercase separated by underscores. Example for a single observatory, such as IceCube is <code>gcn.notices.<I>icecube</i>.<i>lvk_nu_track_search</i></code>. For missions with multiple instruments, you can include the instrument name as <code>gcn.notices.<i>mission</i>.<i>instrument</i>.<i>notices_type</i></code>, for example, Swift-BAT Kafka topic should be <code>gcn.notices.<i>swift</i>.<i>bat</i>.<i>alert</i></code>.
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Review by lpsinger — APPROVED** [review]

**Pull request merged**